### ISSUE TITLE
Feature request: opt-in hide profile and posts from search engines

### ISSUE THREAD

**Issue by dnutiu** [forem/forem#8212]
Hello

**Is your feature request related to a problem? Please describe.**

I'd like to request a privacy wise feature in which an user can request to hide it's account and post from being indexed by search engines.

**Describe the solution you'd like**

A checkbox in the Settings with a text saying something like "Prevent search engines for indexing your profile and posts... you may get less views".

Base on that setting value, a meta tag or header could be generated for every page related to the user:
https://support.google.com/webmasters/answer/93710?hl=en

Thanks!

**Comment by rhymes** [comment]
Hi @dnutiu, I'm going to close this as a duplicate of https://github.com/thepracticaldev/dev.to/issues/3252

Could you add your comment there as well? It'd be awesome!

thank you!

**Comment by dnutiu** [comment]
Hi @rhymes, I'm not sure if this is a duplicate of #3252.

The setting I've described would discourage search engines to index your profile, while #3252 describes a soft deletion of the user's account.

**Comment by rhymes** [comment]
@dnutiu my bad :-) I thought the other one was implying this.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by cmgorton** [comment]
Since this issue has gone stale I am going to close it. If anyone would like to still implement this feature let me know and we can turn this issue into a [discussion](https://github.com/forem/forem/discussions)
